test/e2e_node/e2e_node_suite_test.go

```diff
@@ -54,6 +54,7 @@ var e2es *services.E2EServices
 var runServicesMode = flag.Bool("run-services-mode", false, "If true, only run services (etcd, apiserver) in current process, and not run test.")
 var runKubeletMode = flag.Bool("run-kubelet-mode", false, "If true, only start kubelet, and not run test.")
 var systemValidateMode = flag.Bool("system-validate-mode", false, "If true, only run system validation in current process, and not run test.")
+var enableCriFlag = flag.Bool("enable-cri", false, "If true, test will run in CRI mode and avoid docker-specific assumptions.")
```

Comment:
Sure I will move into there (sorry I didn't noticed it before). Regarding the name change, `enable-cri` was [already referenced](https://github.com/kubernetes/kubernetes/pull/40525/files#diff-19896e63f8ba932b76e0e277a1c4b4beL68) but I'm, not sure about the history/rationale of it. Should I just go ahead and change all references?